# Generated for NaoQi method ALVisionRecognition.run
# protected method

---


